qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -2108,7 +2108,12 @@ class Kubernetes implements OrchestratorMain {
             log.warn("Error while waiting for deployment/populating deployment info: ", e)
         }
         if (!deployment.skipReplicaWait && !deployment.deploymentUid) {
-            throw new OrchestratorManagerException("The deployment did not start or reach replica ready state")
+            String exceptionMsg = "The deployment did not start or reach replica ready state"
+            if (deployment.imagePullPolicy == "Never") {
+                exceptionMsg += " - if this job uses image prefetch check that this image is in the jobs prefetch list"+
```

Comment:
```suggestion
                exceptionMsg += " - if this job uses image prefetch check that this image is in qa-tests-backend/scripts/images-to-prefetch.txt"+
```

Reply:
Ultimately prefetch should run for all jobs. And for the jobs that use .groovy tests there should be a different file per job based on the tests that job runs. 
In hindsight, this file should be something like `qa-tests-backend/image-prefetch/images-for-qa-e2e-all.txt`

Reply:
Sounds like a good idea. But regardless of whether we do this, a pointer to this list, or at least the general location where to find it should really appear in this error message. Otherwise we'll frustrate and slow down the next person who needs to add a new image. 😞